spec.md

```diff
@@ -209,6 +209,18 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### sampled-rate
+* Type: Positive Integer
+* Description: The rate at which this event has already been sampled. Represents
+  the number of similar events that happened but were not sent.
+* Constraints:
+  * OPTIONAL
+  * If present, MUST be a positive integer
+  * When absent means that no sampling has yet been applied
```

Comment:
nit: "yet been applied" is a bit confusing... maybe it will never be applied.  

When events are used for business logic or ETLs, sampling is not really applicable.  For example, user sign-up event or file upload.

Reply:
My intent with that language is to reinforce that intermediary systems may apply additional sampling. How about “When absent, a default value of 1 MAY be assumed, indicating no sampling”? Would love additional phrasing suggestions.